pkg/api/validation/validation.go

```diff
@@ -2784,6 +2788,18 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	return allErrs
 }
 
+// Validate that opaque integer resource quantities in a resource list are
+// indeed integers.
+func ValidateOpaqueIntQuants(rs api.ResourceList, fldPath *field.Path) field.ErrorList {
```

Comment:
So I'm not sure if this would fly with the api-machinery folks mucking the existing resource lists.